.github/workflows/publish-module-builder.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish and release module builder
+
+on:
+  pull_request:
+    types: [ closed ]
```

Comment:
Pretty sure you could do something like this instead of triggering this action everytime a PR is closed.

```yml
on:
  push:
    branches:
      - module-builder-release/v*
```

And you could then remove the `if` on line 10.

Reply:
I think it could work if I was able to specify both the head and base branches, the idea is that we only trigger the build after the PR is merged, but you are correct, maybe it's better just to do that once the Push is done since we are going to open a PR with that branch name when all changes are already at the master branch and we want to release that version.